test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -71,13 +82,41 @@ internal void CreateDotnetNewProject(string solutionRoot, string projectName, st
             Assert.True(string.IsNullOrWhiteSpace(result.Item3), $"Creating project failed with following message in error stream :\n {result.AllOutput}");
         }
 
+        //create a global.json file in temperary testing folder, to make sure testing with the correct sdk when there're multiple of them in CLI folder.
+        internal void CreateTempGlobalJson(string solutionRoot)
```

Comment:
A lot of this logic seems shared. Maybe you can move it to the Test.Utility project?

Reply:
The MsbuildIntegrationTestFixture.CreateTempGlobalJson  needs to check SDK version first(2.2.3 or 3.0.1), then specify a same SDK version in global.json.
However, in Util.CreateTempGlobalJson, it's a temporary workaround since the MSbuild on CI could not resolove the preview SDK version(but we installed SDK3.0 preview under E:\A\_work\_temp\dotnet as the highest version), so we need to specify 3.0.100 - preview in global.json.

Reply:
makes sense. 
Consider this resolved.